kiex

```diff
@@ -503,7 +503,7 @@ function get_known_elixir_releases() {
   x=${x//v/}
 
   # update_release_cache "$x"
-  echo "$x" | sed 's/^/    /' | sort -t. -k 1,1n -k 2,2n -k 3,3n -k 4,4n
+  echo "$x" | sed 's/^/    /' | tac
```

Comment:
This is not compatible with non-Linux operating systems and should not be used. `tac` is a GNUisance.

Reply:
`sort -V` seems to work on macOS.